### PR TITLE
Add blog post "A little encouragement goes a long way" 

### PR DESCRIPTION
**Pull request by jayjayjpg**
Related to the Call for Roadmap Blog Posts https://emberjs.com/blog/2018/05/02/ember-2018-roadmap-call-for-posts.html

screen shot 2018-05-30 at 10 11 10

### CODE REVIEW

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+layout: article
+section: Blog
+title: A Little Encouragement Goes a Long Way in 2018
+author: "Jessica Jordan"
+github-handle: jessica-jordan
+twitter-handle: jjordan_dev
+---
+
+In May 2018, Ember Core team member Katie Gengler published [Ember's 2018 Roadmap: A Call for Blog Posts](https://www.emberjs.com/blog/2018/05/02/ember-2018-roadmap-call-for-posts.html) a call-to-action for the community to give feedback on their hopes and wishes for Ember moving forward.
```
Comment (Turbo87) [inline blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: I would split that sentence after the link and make "a call-to-action [...]" a separate sentence

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+Other productivity improvements that emerged with Ember include its well-planned release cycle ([3](https://www.emberjs.com/blog/2013/09/06/new-ember-release-process.html)) that has been put into practice during the v1.x era as early as 5 years ago, and the ongoing support through LTS releases ([4](https://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html?utm_source=javascriptweekly&utm_medium=email)) since more than 2 years. This kind of matured and dependable release process has not only been extremely progressive back then, but in fact it still is.
+Furthermore, the current testing story in Ember has evolved to be one of the most outstanding community solutions in the JavaScript realm: The modern QUnit Testing API ([5](https://rwjblue.com/2017/10/23/ember-qunit-simplication/)) that ships with v3.x apps out of the box in combination with the simple interface of `@ember-test-helpers` ([6](https://github.com/emberjs/ember-test-helpers/blob/master/API.md)) improves the readability of tests massively and also makes async testing more than straightforward.
```
Comment (Turbo87) [inline blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: `ember-test-helpers` or `@ember/test-helpers` 😉

**Review by Turbo87 — APPROVED** [review]
nice work 👏

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 1 above)
Comment (marcoow) [inline blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: I think there's a comma missing - _"published Ember's 2018 Roadmap: A Call for Blog Posts, a call-to-action…"_

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+---
+
+In May 2018, Ember Core team member Katie Gengler published [Ember's 2018 Roadmap: A Call for Blog Posts](https://www.emberjs.com/blog/2018/05/02/ember-2018-roadmap-call-for-posts.html) a call-to-action for the community to give feedback on their hopes and wishes for Ember moving forward.
+In this context, I also want to share some of my own thoughts on Ember and what I'd be excited to see in its nearest future.
+
+<!--break-->
+
+> The Ember team would like you to write a blog post to propose goals and direction for Ember in the remainder of 2018. The content of these posts will help us to draft our first Roadmap RFC.
+
+First off, I'm confident that Ember already has done a lot to allow me and many others to create great and maintainable applications. And it has done so very early on.
```
Comment (marcoow) [inline blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md:17]: Is _"confident"_ the right word? After all, you **know** that is has done great things I guess? Maybe _"I believe"_ or so?

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+For a while naming conventions and a CLI-driven approach to scaffold, build & serve apps have been hallmarks of Ember as a framework for the ambitious developer. Other framework ecosystems are also adopting similar approaches to increase developer productivity ([1](https://github.com/angular/angular-cli/tree/v1.6.8#cli-for-angular-applications-based-on-the-ember-cli-project), [2](https://hackernoon.com/structuring-projects-and-naming-components-in-react-1261b6e18d76)) and are having immense success with it.
```
Comment (marcoow) [inline blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: I think this should be _"with them"_ as it refers to _"similar approaches"_.

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+Other productivity improvements that emerged with Ember include its well-planned release cycle ([3](https://www.emberjs.com/blog/2013/09/06/new-ember-release-process.html)) that has been put into practice during the v1.x era as early as 5 years ago, and the ongoing support through LTS releases ([4](https://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html?utm_source=javascriptweekly&utm_medium=email)) since more than 2 years. This kind of matured and dependable release process has not only been extremely progressive back then, but in fact it still is.
```
Comment (marcoow) [inline blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: I think instead of _"This kind of matured and dependable…"_ _"This matured and dependable"_ might be better? There's no need to diminish the statement that the release process is in fact matured and dependable.

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+Furthermore, the current testing story in Ember has evolved to be one of the most outstanding community solutions in the JavaScript realm: The modern QUnit Testing API ([5](https://rwjblue.com/2017/10/23/ember-qunit-simplication/)) that ships with v3.x apps out of the box in combination with the simple interface of `@ember-test-helpers` ([6](https://github.com/emberjs/ember-test-helpers/blob/master/API.md)) improves the readability of tests massively and also makes async testing more than straightforward.
+
+These are some of the many examples in which Ember.js has repeatedly proven itself to be a community solution that really cares about its users to get things done.
+This is why Ember is a truly productive JavaScript framework.
+
+Not only in the past has Ember proven to put new ideas into practice in the ecosystem of front-end web development; recent experiments with Webassembly in the Glimmer VM ([7](https://youtu.be/NhtpXs0ZtUc?t=35m56s)) already let us anticipate that Ember.js will continue to run on its innovative trajectory. New features in Ember are in fact often ahead of its time.
```
Comment (marcoow) [inline blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: It's _"WebAssembly"_ with a capital _"A"_.

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+This is why Ember is a truly progressive JavaScript framework.
+
+Progressive technologies give developers a head start in exploring new ways of writing applications. Progressive technologies will draw the interest of many new, early-adopting developers to communities. But progressiveness also poses challenges: depending on their own experience level and the amount of work they spent on critically evaluating a new feature a developer may or may not fully grasp its meaning and extent early on. E.g. a JavaScript developer who entered the field recently, might not be aware that a CLI can help them immensely in building and serving their app yet. Even a person with extensive experience in one certain programming language might not be aware of the challenges they might encounter developing JavaScript applications specifically and overlook the benefits of a progressive feature like Webassembly easily.
```
Comment (marcoow) [inline blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: _"WebAssembly"_ here as well.

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+I believe Ember's bottleneck for adoption and community growth isn't about technical aspects, it's about visibility and outreach.
+
+
+### A Gap in Visibility
+
+As any other open-source project, the development of Ember lives and thrives through the time and amount of work that is being put into it. In contrast to some other major JavaScript frameworks, Ember is not being backed up by one single major company - neither financially nor contribution-wise.
+Instead Ember's user base is diverse in its professional backgrounds and interests. And this makes Ember more independent and free to evolve according to real developer needs instead of company requirements which is an amazing feature.
+
+But who's putting in the time and effort to maintain such an independent project? For a huge part, Ember's community does, and it does so in their free time. Countless contributors work on improvements, bug fixes and features in the early hours preceding their day jobs, on their weekends, their time-off - to bring forward a software project which can compete with other major frameworks. And this in itself is remarkable.
```
Comment (marcoow) [inline blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: this is really not relevant but I think _"…in the early hours preceding their day jobs…"_ is actually _"…in the late hours succeeding their day jobs…"_ for most people :)

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 1 above)
Comment (jayjayjpg) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: Yes, sounds good :+1:

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 2 above)
Comment (jayjayjpg) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: That's right, merci!

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+
+It's not only the Ember Core or the Ember Learning Team who does all of this - it's also you and me and us.
+
+Taking the opportunity to change the way Ember is represented in the world to further its adoption is an open issue everyone can work on.
+Ember is still a somewhat hidden gem ([8](http://blog.agilityworks.co.uk/our-blog/a-hidden-gem-ember-js)) and it keeps on "quietly shipping new cutting-edge features to its users" ([9](https://twitter.com/sugarpirate_/status/923620576970731520)). But I don't believe it has to be this way with a little help from the community.
+
+Of course, there are those who already have a clear vision in mind right now and who are making consistent efforts to create online tutorials, write blog posts, give talks, send pull requests to the Ember.js website repository and much more without any prompting.
```
Comment (marcoow) [inline blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: > …to the Ember.js website repository…

this might be good to link to as well?

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+What I'd wish for Ember's 2018 Roadmap though is to find ways to **lower the entry barriers** for newcomers to get started in their attempt to advocate Ember and to be creative on how to **encourage a sense of empowerment** in the wider community regarding outreach efforts.
+
+### Closing the Visibility Gap
+
+I believe one of the biggest actions that could be taken to make contribution to anything that is outreach-related easier is by increasing the discoverability of discussion channels and already ongoing Github projects and related issues. On the Ember Community Slack chat, most notably the [#st-website](https://embercommunity.slack.com/messages/CAHEZTMBK/), [#wb-marketing](https://embercommunity.slack.com/messages/C36ETE3DK/) and [#topic-talks](https://embercommunity.slack.com/messages/C9RSE508J/) channels are great first points of contact for anyone who was interested in helping out with marketing efforts to find something to help out with. I believe making these channels more discoverable for new developers, e.g. from having a noticeable call-to-action on the `emberjs.com` website or a dedicated, search-engine optimized landing page already goes a long way.
```
Comment (marcoow) [inline blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: > …on the `emberjs.com` website…

should that just be a link instead?

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+
+Also supporting both new and current speakers in finding events, a good talk topic and in prepping and rehearsing a presentation is another great way to encourage the community in their sense of empowerement.
+In fact, the Women Helping Women Program ([16](https://emberwomen.com/)) has already been doing an amazing job in supporting new speakers for years and it might be due to this very program, that Ember managed to achieve record numbers of excellent talks presented by women speakers at its conferences in the past.
+Encouraging even larger parts of the community to speak and highlight Ember allows many great talks to come: I'd love to see so many of the cool and new things that recently landed or are just on its way to land in Ember highlighted: This year's EmberConf Opening Keynote by Ember Core Team members Yehuda Katz & Tom Dale ([17](https://www.youtube.com/watch?v=NhtpXs0ZtUc)) at EmberConf is among others ([18](https://twitter.com/skillsmatter/status/984752827904950273), [19](https://www.youtube.com/watch?v=8D-O4cSteRk)) an inspiring example of this kind of advocacy for Ember.
```
Comment (marcoow) [inline blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: > …on its way…

I think that should be _"on their way"_ as it refers to _"cool and new things"_

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+I feel that beyond that there is a quite urgent need for "introduction" type talks which are appealing to developers of any experience level and any (JavaScript) background. Features of Ember that might have already been around for a while and which have become obvious to the regular Ember user still need to be highlighted to audiences who are not that familiar with Ember yet on a regular basis. Therefore I'm confident supporting speakers in promoting their own "Why Ember" type of presentations to events outside of the Ember ecosystem is one of the most useful measures we as a community can take to advocate Ember as a framework.
+
+### Wishing Upon a Vocal Community
+
+These are some of the ideas that came to my mind when thinking what I'd wish for Ember in 2018. I'd like to see Ember's community grow even further and I believe one major way to achieve this goal, is by improving its visibility - the degree by which it is seen by those who are already becoming part of the community and also by those outside of it.
```
Comment (marcoow) [inline blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: > …when thinking what I'd wish for Ember in 2018.

Is this missing an _"about"_ - _"…when thinking about what I'd wish…"_?

Comment (marcoow) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: > …way to achieve this goal, is by improving…

I'm not sure that comma is actually necessary?

**Review by marcoow — COMMENTED** [review]
This is a really great post - awesome job @jessica-jordan 🙌 

I only left a few comments on grammar, wording etc.

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 3 above)
Comment (geekygrappler) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md:17]: Could just drop it
> First off, Ember has already done a lot to allow me and many others to create great and maintainable applications.

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 5 above)
Comment (geekygrappler) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: **Mature** not _matured_. Either the way it is or with @marcoow version.

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md [hunk 13]
```diff
@@ -0,0 +1,78 @@
...
+For a while naming conventions and a CLI-driven approach to scaffold, build & serve apps have been hallmarks of Ember as a framework for the ambitious developer. Other framework ecosystems are also adopting similar approaches to increase developer productivity ([1](https://github.com/angular/angular-cli/tree/v1.6.8#cli-for-angular-applications-based-on-the-ember-cli-project), [2](https://hackernoon.com/structuring-projects-and-naming-components-in-react-1261b6e18d76)) and are having immense success with it.
+
+Other productivity improvements that emerged with Ember include its well-planned release cycle ([3](https://www.emberjs.com/blog/2013/09/06/new-ember-release-process.html)) that has been put into practice during the v1.x era as early as 5 years ago, and the ongoing support through LTS releases ([4](https://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html?utm_source=javascriptweekly&utm_medium=email)) since more than 2 years. This kind of matured and dependable release process has not only been extremely progressive back then, but in fact it still is.
```
Comment (geekygrappler) [inline blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: > and the ongoing support through LTS release _for_ more than 2 years.

Since doesn't really work in this context.

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md [hunk 14]
```diff
@@ -0,0 +1,78 @@
...
+Furthermore, the current testing story in Ember has evolved to be one of the most outstanding community solutions in the JavaScript realm: The modern QUnit Testing API ([5](https://rwjblue.com/2017/10/23/ember-qunit-simplication/)) that ships with v3.x apps out of the box in combination with the simple interface of `@ember/test-helpers` ([6](https://github.com/emberjs/ember-test-helpers/blob/master/API.md)) improves the readability of tests massively and also makes async testing more than straightforward.
+
+These are some of the many examples in which Ember.js has repeatedly proven itself to be a community solution that really cares about its users to get things done.
```
Comment (geekygrappler) [inline blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: > really cares about getting things done and looking after it's users

or 

>  Really empowers it's users to get things done.

Depending on what you're trying to say. Ember looks after it's users and is a productive framework. Or Ember uses the power of it's community to make it a productive framework.

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 6 above)
Comment (geekygrappler) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: I would say
> recent experiments with WebAssembly in the Glimmer VM **signal** that Ember.js will continue to run on its innovative trajectory.

_already let us anticipate_ doesn't sound quite right.

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 7 above)
Comment (geekygrappler) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: > that a CLI **will** help them immensely in building and serving their app ~yet~.

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md [hunk 15]
```diff
@@ -0,0 +1,78 @@
...
+
+Progressive technologies give developers a head start in exploring new ways of writing applications. Progressive technologies will draw the interest of many new, early-adopting developers to communities. But progressiveness also poses challenges: depending on their own experience level and the amount of work they spent on critically evaluating a new feature a developer may or may not fully grasp its meaning and extent early on. E.g. a JavaScript developer who entered the field recently, might not be aware that a CLI can help them immensely in building and serving their app yet. Even a person with extensive experience in one certain programming language might not be aware of the challenges they might encounter developing JavaScript applications specifically and overlook the benefits of a progressive feature like Webassembly easily.
+
+This makes me think that making features understandable to a diverse set of people does not only include explaining how technologies work, but also why they are so useful.
+
+I already believe that Ember.js is a great choice for any web developer who wants to create scalable applications today.
```
Comment (geekygrappler) [inline blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: >~I already believe that~ Ember.js is a great choice

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md [hunk 16]
```diff
@@ -0,0 +1,78 @@
...
+There's really not much I can wish for on a technical level.
```
Comment (geekygrappler) [inline blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: >I **could** wish for

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md [hunk 17]
```diff
@@ -0,0 +1,78 @@
...
+Instead when I think about what I wish for Ember, _visibility_ comes to my mind.
+I'd love both Ember's core characteristics and its cutting-edge features to be understood by even more people; even by those who haven't had much experience in using it to see its worthwhile benefits in the long-run of a project, yet.
```
Comment (geekygrappler) [inline blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: >~yet~.

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md [hunk 18]
```diff
@@ -0,0 +1,78 @@
...
+I believe Ember's bottleneck for adoption and community growth isn't about technical aspects, it's about visibility and outreach.
+
+
+### A Gap in Visibility
+
+As any other open-source project, the development of Ember lives and thrives through the time and amount of work that is being put into it. In contrast to some other major JavaScript frameworks, Ember is not being backed up by one single major company - neither financially nor contribution-wise.
+Instead Ember's user base is diverse in its professional backgrounds and interests. And this makes Ember more independent and free to evolve according to real developer needs instead of company requirements which is an amazing feature.
```
Comment (geekygrappler) [inline blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: possibly **advantage** over _feature_?

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 3 above)
Comment (jayjayjpg) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md:17]: I like to use these kind of modifiers actually. It helps me to highlight that what I'm saying is still my own personal opinion. Saying that `Ember has done a lot to allow me and many others to create great and maintainable applications` is not a fact, but an interpretation I'm making. Especially for the `others` in this sentence I cannot really speak - I can only assume that they are happy with the framework in a similar way as I am

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 8 above)
Comment (geekygrappler) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: I would reword to
> Ember's community members do, and they do so in their free time.

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 4 above)
Comment (jayjayjpg) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: That's right, I'll update :+1:

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 8 above)
Comment (geekygrappler) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: > on their weekends, **and** their time-off

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 5 above)
Comment (jayjayjpg) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: Yes, that's right - I tried to say `this kind of` as in `this type of` but this doesn't translate well / at all - I'll update both your suggestions :+1:

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md [hunk 19]
```diff
@@ -0,0 +1,78 @@
...
+
+But who's putting in the time and effort to maintain such an independent project? For a huge part, Ember's community does, and it does so in their free time. Countless contributors work on improvements, bug fixes and features in the early hours preceding their day jobs, on their weekends, their time-off - to bring forward a software project which can compete with other major frameworks. And this in itself is remarkable.
+
+This is why Ember truly is a framework from the community for the community.
```
Comment (geekygrappler) [inline blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: Please please please 
> Ember truly is a framework of the community, by the community, for the community... and it shall not perish from the javascript world.

https://en.wikipedia.org/wiki/Gettysburg_Address

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 6 above)
Comment (jayjayjpg) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: Sweet, will update both :+1:

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 3 above)
Comment (marcoow) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md:17]: Yep, that makes sense @jessica-jordan, I just think _"confident"_ has a meaning that does not fit here - you might be confident something will happen or so but I'm not sure it can express the fact that you have the opinion something is already the case?

**Review by wadie — APPROVED** [review]

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 8 above)
Comment (jayjayjpg) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: I think this is good so far, even if I'm not a regular early bird myself :-)

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 9 above)
Comment (jayjayjpg) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: I think in the context of the whole sentence it might make sense to link to the list of pull requests merged into the Ember.js website repo instead of linking to the website itself. I can update it this way

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 10 above)
Comment (jayjayjpg) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: Sure, I'll update to `Ember.js website ([x](link))`

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 13 above)
Comment (jayjayjpg) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: Awesome, merci :+1:

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 15 above)
Comment (jayjayjpg) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: I really like to use the modifier here to underline that it's my personal opinion and everyone else should feel invited to make their own after re-evaluating my arguments

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 16 above)
Comment (jayjayjpg) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: Super, will update :+1:

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 18 above)
Comment (jayjayjpg) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: Yes that sounds pretty suitable, will update

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 8 above)
Comment (marcoow) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: 👍

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 9 above)
Comment (marcoow) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md]: yep, good idea 👍

File: blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md (hunk 3 above)
Comment (jayjayjpg) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md:17]: I think you can use `to be confident` to also state that you believe something is a certain way already in the past / present and not only in the future. See also e.g.:

>confident (comparative more confident, superlative most confident)
>very sure of something; positive
>> I'm pretty confident that she's not lying, she's acting normally.

from [the English Wiktionary article on "confident"](https://en.wiktionary.org/wiki/confident)

Comment (marcoow) [reply blog/_posts/2018-05-30-a-little-encouragement-goes-a-long-way-in-2018.md:17]: ah cool, learned something 👍

**Comment by jayjayjpg** [comment]
@marcoow I updated according to your suggestions, can you take another look and merge if it's alright?

**Review by marcoow — APPROVED** [review]
🎉

**Comment by marcoow** [comment]
online at  https://simplabs.com/blog/2018/05/30/a-little-encouragement-goes-a-long-way-in-2018.html now

**Review by oliverbarnes — COMMENTED** [review]
Hey Jessica, I like the overall message, it's definitely an important topic 👏 

I'd just suggest the text could use some trimming, while keeping the same positive, personal spirit. Saying the same things with less, removing some auxiliaries and some adjectives. It might make it easier to read, and more impactful.

In my humble opinion, as a lousy writer :) it's a lot easier to criticize than to write, that's for sure =D
